symfony/client/src/orangehrmMaintenancePlugin/components/EmployeeRecords.vue

```diff
@@ -40,9 +38,7 @@
           </oxd-grid-item>
         </oxd-grid>
       </oxd-form-row>
-
       <oxd-divider />
-
       <oxd-form-actions>
```

Comment:
I kept blank lines in these places since in some (not all) other components where they used `<oxd-divider />` they had kept blank lines. Not having is also okay

Reply:
just a formatting thing I guess don't think there is a special reason